src/Jackett.Common/Definitions/torrent9.yml

```diff
@@ -8,8 +8,8 @@ encoding: UTF-8
 followredirect: true
 links:
   - https://www.oxtorrent.me/
-  - https://ww1.torrent9.is/
-  - https://ww1.torrent9.to/
+  - https://www.torrent9.is/
```

Comment:
you forgot to add the ww1. links to the legacy section

Reply:
whoops! all sorted